src/reddit.js

```diff
@@ -27,3 +27,5 @@ export async function getCuteUrl() {
   const randomPost = posts[randomIndex];
   return randomPost;
 }
+
+export default { getCuteUrl: getCuteUrl };
```

Comment:
Why make this change?  Generally default exports make things a little less deterministic, and cause a change the second I want to export another thing.  I think this was probably fine the way it was.

Reply:
Usually I don't like to change the code I'm testing but this was the result of a legendary fight with `sinon`. I can revert this and see if I can stub it differently.

Reply:
Went ahead and skipped stubbing getCuteUrl and put your export back. 🤘